tuplex/utils/src/List.cc

```diff
@@ -112,4 +112,14 @@ namespace tuplex {
         }
         return true;
     }
+
+    size_t List::numNonNullElements() const {
+        size_t num = 0;
+        for(size_t i = 0; i < _numElements; i++) {
+            if(!getField(i).isNull()) {
```

Comment:
rewrite maybe into
```
size_t count = 0;
for(size_t i = 0; i < _numElements; i++)
 count += !getField(i).isNull();
```
? potentially we could also iterate directly over the fields, getting rid off the getField(i) function.